Automattic/jetpack
# Podcast Player Block: avoid use of extract()

#### Changes proposed in this Pull Request:

* Avoid the use of `extract()` in the podcast player block's templates.

#### Jetpack product discussion

* p3btAN-1xB-p2
* D67274-code

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check D67274-code and ensure the changes in this PR match.
* Start a new post.
* Insert a podcast player block and load a podcast: https://distributed.blog/category/podcast/feed/
* Preview the post and make sure the block still works as expected.
* Inspect the document source and double-check the markup coming out correctly.
